README.md

```diff
@@ -208,6 +215,9 @@ The following information is required to deploy TSI helm charts:
 * cluster name - name of the cluster. This should correspond to actual name of the cluster
 * cluster region - label associated with the actual region for the data center (e.g. eu-de, dal09, wdc01)
 * vault address - the remote address of the Vault service that contains the TSI secrets to be retrieved by the sidecar. Use the env. variable VAULT_ADDR set [above](./README.md#setup-vault)
+* jss service - TSI currently support 2 mechanism for running the JSS (JWT Signing Service):
+  - jss-server - custom service for signing JWT tokens (default)
+  - vtpm2-server - JWT token signer using a software wrapper for TPM2
```

Comment:
Can we add the keys to actually ovewrite for each one here, since they have different paths?

like jss service (jssService.type),
cluster region (ti-key-release-1.cluster.region)

Reply:
There is no easy way to assign values to variables in Helm.